prez/routers/management.py

```diff
@@ -12,23 +13,29 @@
 from prez.cache import endpoints_graph_cache
 from prez.config import settings
 from prez.reference_data.prez_ns import PREZ
-from prez.renderers.renderer import return_rdf
-from prez.services.connegp_service import RDF_MEDIATYPES
+from prez.renderers.renderer import return_rdf, return_from_graph
+from prez.services.connegp_service import RDF_MEDIATYPES, MediaType
 
 router = APIRouter(tags=["Management"])
 log = logging.getLogger(__name__)
 
 
 @router.get("/", summary="Home page", tags=["Prez"])
-async def index():
+async def index(
+        mediatype: Optional[MediaType] = Query(
+            MediaType("text/turtle"), alias="_mediatype", description="Requested mediatype"
+        ),
+):
     """Returns the following information about the API"""
+    if "anot+" in mediatype.value:
+        mediatype = MediaType(mediatype.value.replace("anot+", ""))
     g = Graph()
     g.bind("prez", "https://prez.dev/")
     g.bind("ont", "https://prez.dev/ont/")
     g.add((URIRef(settings.system_uri), PREZ.version, Literal(settings.prez_version)))
     g += endpoints_graph_cache
     g += await return_annotation_predicates()
-    return await return_rdf(g, "text/turtle", profile_headers={})
+    return await return_rdf(g, mediatype.value, profile_headers={})
```

Comment:
I also noticed as part of this that the return_rdf function https://github.com/RDFLib/prez/blob/78fda6864d1e5593b9844bf35e9d86ac14024c2b/prez/renderers/renderer.py#L84

potentially needs improvement to be more robust. If it recieves an anotated media type other than `text/anot+turtle` it will cause an error because it won't find it in the RDF_MEDIATYPES dictionary.

Reply:
@recalcitrantsupplant does this let you view it?

Reply:
Thanks, I've switched it to use connegp and added a change to use the return_from_graph function which will handle the other annotated mediatypes. I think these functions in general need a cleanup.

https://github.com/RDFLib/prez/pull/246/commits/eba841126174ccbc3cd1928c1693687e2e92ca06